### DISCUSSION TITLE
"Oops, your response got filtered" on C# ASP.NET code?

### DISCUSSION THREAD

**Discussion by sapalazz** [community/community#110609] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Morning everyone. 

I've been using Copilot for a week or so, but always in a React repo.
I started "using" it today on a C# repo and this is what I'm getting. Not sure if it's a bug, or if it's incompatible with this? I'm using a Microsoft Remote Desktop and in the React case, just the Mac.

Just giving some ideas in case anyone has a clue.

Thank you :) 

![Captura de pantalla 2024-03-01 a las 10 46 23](https://github.com/community/community/assets/71673769/315bfc66-9ae2-4d6a-93b6-b5e41cf4c8d6)

**Comment by Akash1134** [comment, marked as answer]
Hi @sapalazz , Please update your IDE and that should fix this issue for you. Refer this [issue reported](https://github.com/microsoft/vscode-copilot-release/issues/822#issuecomment-1924064375) that got resolved after updating IDE - let us know how it goes.

Thanks!

**Reply by sapalazz** [reply]
Yes! It has finally worked. Thank you! Have a great day. 🥇
